.github/workflows/ci.yml

```diff
@@ -35,11 +35,15 @@ jobs:
         env:
           discovery.type: single-node
           plugins.security.disabled: true
+          OPENSEARCH_JAVA_OPTS: -Xms512m -Xmx512m
+          DISABLE_INSTALL_DEMO_CONFIG: true
       opensearch2:
         image: opensearchproject/opensearch:2.13.0
         env:
           discovery.type: single-node
           plugins.security.disabled: true
+          OPENSEARCH_JAVA_OPTS: -Xms512m -Xmx512m
+          DISABLE_INSTALL_DEMO_CONFIG: true
           OPENSEARCH_INITIAL_ADMIN_PASSWORD: temba
```

Comment:
Followed the docker compose file for development
https://opensearch.org/docs/1.3/install-and-configure/install-opensearch/docker/#sample-docker-compose-file-for-development